doc/Changelog.md

```diff
@@ -2,6 +2,8 @@
 
 # git master
 
+* [10](https://github.com/BlueBrain/Keyv/pull/10):
+  Tune memcached behaviour parameters
```

Comment:
I'm really getting lost as to where document stuff. The poor end user does not give a pork. It is documented for the poor developer in the code.